### DISCUSSION TITLE
yeni avax  mı sizce

### DISCUSSION THREAD

**Discussion by Coinemo** [IBAX-io/go-ibax#1292] [Bounty Hunter]
neden yeni avax diye lanse ediliyor ? avax ile karşılaştırılacak kadar güçlü bir altyapısı var mı ?

**Comment by ahmederoni** [comment]
Bu konu için bende meraklıyım 5-6 ay veya seneye bu vakitler görecez

**Reply by btcnapolyon** [reply]
Her şeyden önce kendi ağlarını oluşturuyorlar bir token olmayacak yani ve genel olarak beklenti o yönde.

**Comment by Skips1** [comment]
Bunu söylemek için çok erken kanka

**Comment by ysfcndgr** [comment]
Arkadaşlar burası bug bounty bölgesi. Buradaki konularınız sistemlerimizde bulduğunuz güvenlik açıklarıyla alakalı olmalıdır.
